spec/acceptance/consumer_spec.rb

```diff
@@ -140,15 +140,6 @@ class { 'kafka::consumer':
         apply_manifest(pp, catch_failures: true)
       end
 
-      describe file('/etc/init.d/kafka-consumer'), if: (fact('service_provider') == 'upstart' && fact('osfamily') == 'Debian') do
-        it { is_expected.to be_file }
-        it { is_expected.to be_owned_by 'root' }
-        it { is_expected.to be_grouped_into 'root' }
-        it { is_expected.to contain %r{^# Provides:\s+kafka-consumer$} }
-        it { is_expected.to contain 'export KAFKA_JMX_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.port=9993"' }
-        it { is_expected.to contain 'export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:/opt/kafka/config/log4j.properties"' }
-      end
-
       describe file('/etc/systemd/system/kafka-consumer.service'), if: (fact('operatingsystemmajrelease') == '7' && fact('osfamily') == 'RedHat') do
```

Comment:
Would this now be true for any OS? And shouldn't this have been modified when EL 8 & EL 9 support was added?

Reply:
looking back at this, i looked at the wrong line. And yes, this maybe should have been modified some time ago and now it should be a check for all our supported OSes